src/components/Settings/ToggleFullWidth.tsx

```diff
@@ -0,0 +1,53 @@
+import { useTranslation } from 'next-i18next';
+
+import classNames from 'classnames';
+
+import { Translation } from '@/src/types/translation';
+
+interface ToggleFullWidthProps {
+  isOn: boolean;
+  handleSwitch: () => void;
+}
+
+interface SwitchStateTextProps {
+  switchText: string;
+}
+
+const SwitchStateText = ({ switchText }: SwitchStateTextProps) => (
+  <span className="h-[15px] w-6 text-xs">{switchText}</span>
+);
+
+export const ToggleFullWidth = ({
+  isOn,
+  handleSwitch,
+}: ToggleFullWidthProps) => {
+  const { t } = useTranslation(Translation.Settings);
+
+  const switchBackground = isOn
+    ? 'bg-blue-500 dark:bg-blue-500'
+    : 'bg-gray-400 dark:bg-gray-600';
+
+  return (
+    <label htmlFor="toggle" className="flex w-full items-center gap-5">
+      <span className="w-[120px]">{t('Full width chat')}</span>
```

Comment:
Why do you need to have fixed width here?

Reply:
Fixed width was needed to be alined with design
Changed to use flex values.